test/extended/README.md

```diff
@@ -33,6 +36,60 @@ $ go get github.com/onsi/ginkgo/ginkgo
 You also need to have the `openshift` binary in the `PATH` if you want to use
 the shell script helpers to execute the extended tests.
 
+Rapid local testing
+--------------------
+
+If you already have a running OpenShift cluster, e.g. one created using `oc
+cluster up`, you can skip having the extended test infrastructure spin up an
+OpenShift cluster each time the tests are run by setting the `TEST_ONLY`
+environment variable as follows:
+
+```console
+$ oc cluster up
+$ oc login -u system:admin
+```
+
+Then, for example:
+```console
+$ make build-extended-test
+$ TEST_ONLY=1 test/extended/core.sh --ginkgo.focus='\[builds\]'
+```
+
+By default the Kubernetes test framework will remove the project associated with
+your test spec when it completes, regardless of whether it fails or not.  This
+can be inconvenient when debugging.  To stop this behaviour, add the argument
+--delete-namespace=false:
+
+```console
+$ make build-extended-test
+$ TEST_ONLY=1 test/extended/core.sh --delete-namespace=false --ginkgo.focus='\[builds\]'
+```
+
+Test labels
+-----------
+
+See [kinds of tests](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md#kinds-of-tests)
+for a full explanation of the labels used for each test spec.  In brief:
+
+- If a test has no labels, it is expected to run fast (under five minutes), be
+  able to be run in parallel, and be consistent.
+
+- \[Serial\]: If a test cannot be run in parallel with other tests (e.g. it
+	takes too many resources or restarts nodes), it is labeled \[Serial\], and
+	should be run in serial as part of a separate suite.
+
+- \[Slow\]: If a test takes more than five minutes to run (by itself or in
+	parallel with many other tests), it is labeled \[Slow\]. This partition allows
+	us to run almost all of our tests quickly in parallel, without waiting for the
+	stragglers to finish.
+
+	OpenShift extended tests that run builds should be marked\[Slow\].
+
+- \[Conformance\] tests represent a subset of the tests we expect to pass on any
+  Kubernetes cluster. The \[Conformance\] label does not supersede any other
+	labels.  For more details see [conformance
+	tests](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md#conformance-tests).
```

Comment:
I think this definition might not be accurate.  defer to @smarterclayton for what the requirements are for a test to be labeled "conformance".

(I think the condition is more like "provides unique and critical coverage")